spec.html

```diff
@@ -38775,7 +38778,10 @@ <h1>%AsyncFromSyncIteratorPrototype%.return ( _value_ )</h1>
               1. Let _iterResult_ be ! CreateIterResultObject(_value_, *true*).
               1. Perform ! Call(_promiseCapability_.[[Resolve]], *undefined*, &laquo; _iterResult_ &raquo;).
               1. Return _promiseCapability_.[[Promise]].
-            1. Let _result_ be Call(_return_, _syncIterator_, &laquo; _value_ &raquo;).
+            1. If _value_ is present, then
+              1. Let _result_ be Call(_return_, _syncIterator_, &laquo; _value_ &raquo;).
+            1. Else,
+              1. Let _result_ be Call(_return_, _syncIterator_, &laquo; &raquo;).
```

Comment:
Last argument of [`Call`](https://tc39.es/ecma262/#sec-call) defaults to empty list.
Please also see https://github.com/tc39/ecma262/pull/1414.